sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_responses.hpp

```diff
@@ -176,7 +176,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Azure::Core::Nullable<std::string> Continuation;
   };
 
-  using DirectorySetAccessControlRecursiveInfo = PathSetAccessControlRecursiveResult;
+  using SetDirectoryAccessControlRecursiveResult = PathSetAccessControlRecursiveResult;
```

Comment:
Is this a breaking change?

Reply:
No, since this is added in this release for usage. The previous release only has this definition that customer has no use of.